remappers/hostmetrics/process.go

```diff
@@ -277,5 +277,9 @@ func addProcessResources(resource pcommon.Resource) func(pmetric.NumberDataPoint
 		if name.Str() != "" {
 			dp.Attributes().PutStr("process.name", name.Str())
 		}
+		cmdline, _ := resource.Attributes().Get("process.command_line")
+		if cmdline.Str() != "" {
+			dp.Attributes().PutStr("system.process.cmdline", cmdline.Str())
+		}
```

Comment:
I thought we were leaving the mapping of resource attributes to the callers (elatic processor, MIS/APM), is this not the case? I was under the impression that we would be getting rid of the whole of `addProcessResource`

Reply:
Yes, that's the plan. As of now, this doesn't exist in Exporter. Especially, handling the resource attributes whose names are different in OTEL and Elastic. Until then will keep it here. Once it is addressed in Exporter, will be removing the addProcessResource. 
The `system.process.cmdline` is being used in the Processes tab of the Host UI. Hence its important to have this.